nl_server/loader.py

```diff
@@ -107,15 +127,15 @@ def _load_yaml(flask_env: str) -> Dict[str, any]:
   return embeddings_map
 
 
-def _update_app_config(app: Flask,
-                       attribute_model: NLAttributeModel,
+def _update_app_config(app: Flask, attribute_model: NLAttributeModel,
                        nl_embeddings: emb_map.EmbeddingsMap,
                        embeddings_version_map: Dict[str, any],
-                       vertex_ai_models: Dict[str, Dict] = None):
+                       embeddings_spec: EmbeddingsSpec):
   app.config[config.ATTRIBUTE_MODEL_KEY] = attribute_model
   app.config[config.NL_EMBEDDINGS_KEY] = nl_embeddings
   app.config[config.NL_EMBEDDINGS_VERSION_KEY] = embeddings_version_map
-  app.config[config.VERTEX_AI_MODELS_KEY] = vertex_ai_models or {}
+  app.config[config.DEFAULT_INDEX_TYPE_KEY] = embeddings_spec.default_index
```

Comment:
nit: should we just get the default index from app.config[EMBEDDINGS_SPEC_KEY]?

Reply:
done